Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/type/CoronaTest.kt

```diff
@@ -49,10 +29,16 @@ interface CoronaTest : Recyclable {
     // Has the corresponding entry been created in the test certificate storage
     val isDccDataSetCreated: Boolean
 
-    //  The ID of the lab that uploaded the test result
-    val labId: String?
-
     val qrCodeHash: String?
+}
+
+interface KeySubmission {
```

Comment:
Maybe we should be more specific here as it is for test `CoronaTestKeySubmission` or `CoronaTestSubmission` ?